src/services/api.tsx

```diff
@@ -53,14 +53,14 @@ const useGetUrlsQuery = (userId: string, options: { enabled: boolean }) => {
 
 const useShortenUrlMutation = () => {
     return useMutation(
-        async ({ originalUrl, userData }: { originalUrl: string; userData: UserTypes }) => {
+        async ({ originalUrl, userData }: { originalUrl: string; userData: User }) => {
             const response = await axios.post(
                 `${TINY_API_URL}/tinyurl`,
                 {
                     OriginalUrl: originalUrl,
                     Comment: '',
-                    CreatedBy: userData?.userName,
-                    UserId: userData?.id,
+                    CreatedBy: userData?.data?.userName,
+                    UserId: userData?.data?.id,
```

Comment:
suggestion
---
should we decouple the api with `useMutation` hook so that if we want to invalidate different type of queries after this mutation it can be done with the need for `useShortenUrlMutation` hook to take the query key as a prop.

Reply:
@sahsisunny Please have a look over this